spec/features/staff_can_update_a_scheme_spec.rb

```diff
@@ -25,6 +25,8 @@
       fill_in 'scheme[contractor_email_address]', with: 'new@email.com'
       fill_in 'scheme[employer_agent_name]', with: 'Alex'
       fill_in 'scheme[employer_agent_email_address]', with: 'alex@example.com'
+      fill_in 'scheme[project_manager_name]', with: 'Sara'
+      fill_in 'scheme[project_manager_email_address]', with: 'sara@example.com'
```

Comment:
are we missing `employer_agent_phone_number` from this description of how the scheme can be updated?

Reply:
Good spot, will add!